src/pet.js

```diff
@@ -0,0 +1,22 @@
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+}
+Pet.prototype.growUp = function () { 
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
+};
+
+Pet.prototype.walk = function () {
+   if ((this.fitness + 4) <= 10) {
+    this.fitness += 4;
+   } else {
+    this.fitness = 10;
+   }
+};
```

Comment:
I mentioned about indentation in the previous file, the same issue is showing here. Most of the file uses 4 spaces in, yet here there is only one on lines 15 and 17. As said before whichever level you decide to use it should be made uniform across the whole project